lesson_2.ipynb

```diff
@@ -0,0 +1,2129 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ### HIDE THIS CODE
</pre><p>Great idea</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='109'/>